### ISSUE TITLE
AttributeError: 'Script' object has no attribute 'generate_button'

### ISSUE THREAD

**Issue by hunssandwich** [ArtVentureX/sd-webui-agent-scheduler#15]
tried both built-in extension and Manual clone

**Comment by AimVaguely** [comment]
This is caused by https://github.com/ArtVentureX/sd-webui-agent-scheduler/commit/b92fde92f4f224f3cff817859eb3a5e84b0e7bb2. I think it is actually a typo in the code. 

The workaround/fix is to remove line 26 (`self.generate_button`) in `sd-webui-agent-scheduler/scripts/task_scheduler.py`

**Comment by AimVaguely** [comment]
This might be a duplicate of https://github.com/vladmandic/automatic/issues/1293, but that issue was closed for being a duplicate of something that looks unrelated.

**Comment by Zyin055** [comment]
Got the same error after updating today. Worked fine when it was initially installed 1-2 days ago.

```
Closing server running on port: 7860
Restarting UI...
Civitai Helper: Get Custom Model Folder
Civitai Helper: Load setting from: C:\Stuff\AI\Stable Diffusion\extensions\Stable-Diffusion-Webui-Civitai-Helper\setting.json
Civitai Helper: No setting file, use default
Additional Network extension not installed, Only hijack built-in lora
LoCon Extension hijack built-in lora successfully
[-] ADetailer initialized. version: 23.6.1.post0, num models: 8
[AddNet] Updating model hashes...
0it [00:00, ?it/s]
2023-06-03 22:18:32,546 - ControlNet - INFO - ControlNet v1.1.215
ControlNet v1.1.215
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ C:\Stuff\AI\Stable Diffusion\launch.py:370 in <module>                                           │
│                                                                                                  │
│   367                                                                                            │
│   368 if __name__ == "__main__":                                                                 │
│   369 │   prepare_environment()                                                                  │
│ ❱ 370 │   start()                                                                                │
│   371                                                                                            │
│                                                                                                  │
│ C:\Stuff\AI\Stable Diffusion\launch.py:365 in start                                              │
│                                                                                                  │
│   362 │   if '--nowebui' in sys.argv:                                                            │
│   363 │   │   webui.api_only()                                                                   │
│   364 │   else:                                                                                  │
│ ❱ 365 │   │   webui.webui()                                                                      │
│   366                                                                                            │
│   367                                                                                            │
│   368 if __name__ == "__main__":                                                                 │
│                                                                                                  │
│ C:\Stuff\AI\Stable Diffusion\webui.py:295 in webui                                               │
│                                                                                                  │
│   292 │   │   modules.script_callbacks.before_ui_callback()                                      │
│   293 │   │   startup_timer.record("scripts before_ui_callback")                                 │
│   294 │   │                                                                                      │
│ ❱ 295 │   │   shared.demo = modules.ui.create_ui()                                               │
│   296 │   │   startup_timer.record("create ui")                                                  │
│   297 │   │                                                                                      │
│   298 │   │   if not cmd_opts.no_gradio_queue:                                                   │
│                                                                                                  │
│ C:\Stuff\AI\Stable Diffusion\modules\ui.py:461 in create_ui                                      │
│                                                                                                  │
│    458 │   parameters_copypaste.reset()                                                          │
│    459 │                                                                                         │
│    460 │   modules.scripts.scripts_current = modules.scripts.scripts_txt2img                     │
│ ❱  461 │   modules.scripts.scripts_txt2img.initialize_scripts(is_img2img=False)                  │
│    462 │                                                                                         │
│    463 │   with gr.Blocks(analytics_enabled=False) as txt2img_interface:                         │
│    464 │   │   txt2img_prompt, txt2img_prompt_styles, txt2img_negative_prompt, submit, _, _, tx  │
│                                                                                                  │
│ C:\Stuff\AI\Stable Diffusion\modules\scripts.py:299 in initialize_scripts                        │
│                                                                                                  │
│   296 │   │   auto_processing_scripts = scripts_auto_postprocessing.create_auto_preprocessing_   │
│   297 │   │                                                                                      │
│   298 │   │   for script_class, path, basedir, script_module in auto_processing_scripts + scri   │
│ ❱ 299 │   │   │   script = script_class()                                                        │
│   300 │   │   │   script.filename = path                                                         │
│   301 │   │   │   script.is_txt2img = not is_img2img                                             │
│   302 │   │   │   script.is_img2img = is_img2img                                                 │
│                                                                                                  │
│ C:\Stuff\AI\Stable Diffusion\extensions\sd-webui-agent-scheduler\scripts\task_scheduler.py:26 in │
│ __init__                                                                                         │
│                                                                                                  │
│    23 │   │   super().__init__()                                                                 │
│    24 │   │   script_callbacks.on_app_started(lambda block, _: self.on_app_started(block))       │
│    25 │   │   self.checkpoint_override = checkpoint_current                                      │
│ ❱  26 │   │   self.generate_button                                                               │
│    27 │                                                                                          │
│    28 │   def title(self):                                                                       │
│    29 │   │   return "Agent Scheduler"                                                           │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
AttributeError: 'Script' object has no attribute 'generate_button'
Press any key to continue . . .
```

**Comment by artventurdev** [comment]
@Zyin055 Can you verify if the issue is fixed after #14

**Comment by Zyin055** [comment]
That fixed it
